spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -10,31 +10,31 @@
       Session.create!(
         start: 3.days.ago,
         username: username,
-        mac: '',
-        ap: '',
-        siteIP: '1.1.1.1',
+        mac: "",
```

Comment:
We really need to establish a convention around single vs double quotes.

Reply:
Agreed. I almost didn't include this in the pull request, it annoyed me both ways!

I don't care apart from consistency. Do you have a preference? iirc @tommotaylor slightly preferred double-quotes, but don't quote me on that.

Flip a coin then add a custom rubocop?